scripts/agent.py

```diff
@@ -80,6 +80,111 @@ class RunSQLReturnPandas(BaseModel):
     )
 
     async def execute(self, conn: Connection, limit: int):
+        prompt = f"""
+        ```markdown
+        You are a SQL expert tasked with writing queries including a time attribute for the relevant table. The user wants to execute a query for the following repos: {self.repos} to answer the query of {self.query}. 
```

Comment:
If we use self.query, it generates sql sometimes. A way we can mititigate this is by calling it something else like `query_description`